Validation: Duplicate alt text and duplicate description error fixes. (#1420)
dotnet/docs-desktop: dotnet-desktop-guide/framework/winforms/advanced/antialiasing-with-lines-and-curves.md

---
title: "Antialiasing with Lines and Curves"

> description: Learn how GDI+ works with the display driver software to apply antialiasing to pixels within lines and curves.

Reviewer: ```suggestion
description: Learn how to apply antialiasing to a line and curve in GDI+ for Windows Forms.
```
